src/main/java/frc/robot/Subsystems/ElevatorSubsystem.java

```diff
@@ -1,56 +1,63 @@
-
 package frc.robot.Subsystems;
 
-import java.util.ArrayList;
-import java.util.List;
-
 import com.revrobotics.CANSparkMax;
-import com.revrobotics.RelativeEncoder;
-import com.revrobotics.SparkMaxPIDController;
 import com.revrobotics.CANSparkMax.IdleMode;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
-
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ElevatorConstants;
 
-import frc.robot.Constants;
 
 //TO DO: error handling for if pid controllers get misaligned
 public class ElevatorSubsystem extends SubsystemBase {
+
     private final CANSparkMax elevatorMotor1;
     private final CANSparkMax elevatorMotor2;
-   
+
     private final SparkMaxPIDController m_pidController1;
     private final RelativeEncoder m_encoder1;
     private final RelativeEncoder m_encoder2;
-    public static double kP, kI, kD, kIz, kFF, kMaxOutput, kMinOutput, elevatorSpeed;
+    public static double kP, kI, kD, kIz, kFF, kMaxOutput, kMinOutput, elevatorSpeed, allowedErr;
+    //smart motion constants
+    public static double maxVel, maxAccel, minVel; // rpm
```

Comment:
If we're not using smart motion then we should delete this code and any code related to it.